### PR TITLE
PE-2309: fix: previous versions cannot be downloaded in sharing landing page mode

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
If we access a shared file from a private drive and paste it in a new tab, it is not possible to download any revision of the file from the 'Details' and 'Activity' panel.

Now we are using the `FileRevision` to download the file.

--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/4b2dr4krpceh8
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/5a2i81bjemppg

### CODE REVIEW

File: lib/pages/profile_auth/components/profile_auth_prompt_wallet_screen.dart [hunk 1]
```diff
@@ -64,6 +65,7 @@ class ProfileAuthPromptWalletScreen extends StatelessWidget {
                 textAlign: TextAlign.center,
               ),
             ),
+            // const Spacer(),
```
Comment (matibat) [inline lib/pages/profile_auth/components/profile_auth_prompt_wallet_screen.dart:68]: Nit: unnecessary comment line

**Review by matibat — APPROVED** [review]
The code LGTM - I haven't yet manually tested the feature

**Comment by matibat** [comment]
I tested on Safari. It does work 💯

File: lib/blocs/file_download/file_download_cubit.dart [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import 'dart:async';
+import 'dart:developer';
```
Comment (karlprieb) [inline lib/blocs/file_download/file_download_cubit.dart:2]: Why do we need this package?

**Review by karlprieb — APPROVED** [review]

File: lib/blocs/file_download/file_download_cubit.dart (hunk 2 above)
Comment (thiagocarvalhodev) [reply lib/blocs/file_download/file_download_cubit.dart:2]: Oh, that's to use the `log` function to avoid using prints

**Pull request merged**